docs/logs/perf_N_neighbors_physical_2020_12_10.log

```diff
@@ -0,0 +1,97 @@
+Number: 93 Resource Type: 3 with id: 93-baad-f00d-4444-555555555555 took: 30666 microseconds or 30 milliseconds
+Number: 94 Resource Type: 3 with id: 94-baad-f00d-4444-555555555555 took: 29816 microseconds or 29 milliseconds
+Number: 95 Resource Type: 3 with id: 92-baad-f00d-4444-555555555555 took: 37323 microseconds or 37 milliseconds
+Number: 96 Resource Type: 3 with id: 96-baad-f00d-4444-555555555555 took: 21952 microseconds or 21 milliseconds
+Number: 97 Resource Type: 3 with id: 95-baad-f00d-4444-555555555555 took: 30628 microseconds or 30 milliseconds
+Number: 98 Resource Type: 3 with id: 99-baad-f00d-4444-555555555555 took: 19801 microseconds or 19 milliseconds
+Number: 99 Resource Type: 3 with id: 98-baad-f00d-4444-555555555555 took: 21117 microseconds or 21 milliseconds
+Number: 100 Resource Type: 3 with id: 97-baad-f00d-4444-555555555555 took: 24923 microseconds or 24 milliseconds
+Average port + neighbor states with NeighborType: 0 Create of 100 took: 40295 microseconds or 40 milliseconds
+[TEST METRICS] Elapsed time for message total operation took: 244189 microseconds or 244 milliseconds
+
+g_total_execute_system_time = 3482327 microseconds or 3482 milliseconds
+g_total_execute_ovsdb_time = 1593966 microseconds or 1593 milliseconds
+g_total_execute_openflow_time = 2034118 microseconds or 2034 milliseconds
+g_total_vpcs_table_mutex_time = 0 microseconds or 0 milliseconds
+g_total_update_GS_time = 244189 microseconds or 244 milliseconds
+
+[***1 port + 1000 neighbor***]
+Number: 991 Resource Type: 3 with id: 987-baad-f00d-4444-555555555555 took: 184892 microseconds or 184 milliseconds
+Number: 992 Resource Type: 3 with id: 989-baad-f00d-4444-555555555555 took: 148205 microseconds or 148 milliseconds
+Number: 993 Resource Type: 3 with id: 994-baad-f00d-4444-555555555555 took: 143575 microseconds or 143 milliseconds
+Number: 994 Resource Type: 3 with id: 985-baad-f00d-4444-555555555555 took: 189367 microseconds or 189 milliseconds
+Number: 995 Resource Type: 3 with id: 996-baad-f00d-4444-555555555555 took: 114374 microseconds or 114 milliseconds
+Number: 996 Resource Type: 3 with id: 995-baad-f00d-4444-555555555555 took: 127072 microseconds or 127 milliseconds
+Number: 997 Resource Type: 3 with id: 993-baad-f00d-4444-555555555555 took: 155519 microseconds or 155 milliseconds
+Number: 998 Resource Type: 3 with id: 998-baad-f00d-4444-555555555555 took: 55364 microseconds or 55 milliseconds
+Number: 999 Resource Type: 3 with id: 997-baad-f00d-4444-555555555555 took: 92841 microseconds or 92 milliseconds
+Number: 1000 Resource Type: 3 with id: 999-baad-f00d-4444-555555555555 took: 45760 microseconds or 45 milliseconds
+Average port + neighbor states with NeighborType: 0 Create of 1000 took: 111029 microseconds or 111 milliseconds
+[TEST METRICS] Elapsed time for message total operation took: 7653970 microseconds or 7653 milliseconds
```

Comment:
This number for 1K neighbor cuts the latency quite significantly. Let us keep pushing it to 10K as well.

Reply:
yes, I have a few ideas in mind, like:
1. switch to James's aca_ovs_control.add_flow call instead of executing system calls. aca_ovs_control.add_flow requires some troubleshooting/fixing which I will look into.
1. bundling with neighbor info for ACA processing and aca_ovs_control.add_flow

Reply:
Cool. Let us create two separate tracking issues to follow up.

Reply:
Yes, the first issue is tracked by (4) in #120, I will look into that next.

For the second issue, I opened #178